pyproject.toml

```diff
@@ -6,5 +9,9 @@ line_length = 120
 multi_line_output = 3
 profile = "black"
 
-[tool.black]
-line-length = 120
+[tool.mypy]
+ignore_missing_imports = true
+strict = true
```

Comment:
`mypy` settings are supported in `pyproject.toml` now.

Reply:
is there a specific version of mypy that we need to enforce, to be able to use this functionality?

Reply:
Do we use `mypy` anywhere other than: https://github.com/argoai/argoverse-api/blob/ceaee0f0e09281fc36f58900388a0d00fd6841f9/tox.ini?